packages/api/src/brevo-api/brevo-api-contact.service.ts

```diff
@@ -95,8 +110,8 @@ export class BrevoApiContactsService {
             if (!contact) return undefined;
             return contact;
         } catch (error) {
-            // Brevo throws 400 error if no contact was found
-            if (isErrorFromBrevo(error) && error.response.statusCode === 400) {
+            // Brevo throws 404 error if no contact was found
+            if (isErrorFromBrevo(error) && error.response.statusCode === 404) {
```

Comment:
why is this changed? In the main branch thomas changed that to catch both errors:
https://github.com/vivid-planet/comet-brevo-module/pull/94/files

Reply:
I had problems, as I got a 404 error and I thought that the wrong code was given in the code. I checked it out at Brevo now, and I agree, that both codes should be handled. Thanks for fixing it and pointing it out here as well. 👍